### PR TITLE
Feature/cleanup

### PR DESCRIPTION
**Pull request by tbrezot**

### CODE REVIEW

**Review by ThibaudDauce — DISMISSED** [review]
Awesome job!

File: Cargo.lock [hunk 1]
```diff
@@ -77,9 +77,9 @@ dependencies = [
 
 [[package]]
 name = "bumpalo"
-version = "3.11.1"
+version = "3.12.0"
```
Comment (Manuthor) [inline Cargo.lock:80]: Removed Cargo.lock as it is not recommended for a library: see answer of `alexcrichton` https://github.com/rust-lang/cargo/issues/315#issuecomment-51075730

This way, dependencies are not locked and are not scanned/reported by Github Security Alert

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -17,27 +17,12 @@ jobs:
     with:
       toolchain: stable
       publish: false
-  wasm:
```
Comment (Manuthor) [inline .github/workflows/ci.yml:20]: Is there any risk to discover later (in project where `crypto_core` is required) than `crypto_core` is not wasm32 compliant?

Comment (tbrezot) [reply .github/workflows/ci.yml:20]: Indeed, needed to  keep the specification of the `getrandom` feature in the `Cargo.toml`

File: Cargo.toml [hunk 3]
```diff
@@ -14,13 +14,11 @@ path = "src/lib.rs"
 [dependencies]
 aes-gcm = "0.10"
 curve25519-dalek = "3.2"
-getrandom = { version = "0.2", features = ["js"] }
-hex = "0.4"
+# needed to compile into WASM
+getrandom = { version ="0.2.8", features = ["js"] }
```
Comment (Manuthor) [inline Cargo.toml:18]:
```suggestion
getrandom = { version ="0.2", features = ["js"] }
```

**Review by Manuthor — APPROVED** [review]

**Review by Manuthor — APPROVED** [review]

**Pull request merged**